packages/babel-plugin-jest-hoist/src/index.js

```diff
@@ -71,7 +71,7 @@ const WHITELISTED_IDENTIFIERS = {
 };
 Object.keys(global).forEach(name => (WHITELISTED_IDENTIFIERS[name] = true));
 
-const JEST_GLOBAL = {name: 'jest'};
+const JEST_GLOBAL = { name: 'jest' };
```

Comment:
I think your IDE autoformatted?

Reply:
Hmm, I guess is this one https://github.com/facebook/jest/blob/master/.vscode/settings.json#L8
Reverted.